CMakeLists.txt

```diff
@@ -194,7 +194,7 @@ if(ENABLE_IPV6 AND NOT WIN32)
   endif()
 endif()
 
-option(ENABLE_MANUAL "to provide the built-in manual" ON)
+option(ENABLE_MANUAL "to provide the built-in manual and docs" ON)
```

Comment:
I think this change is unnecessary because ENABLE_MANUAL only builds the manual (from the docs folder) and nothing else